imports/ui/unit/unit.jsx

```diff
@@ -83,12 +114,103 @@ class Unit extends Component {
       })
     }
   }
+  handleReportFilterClicked = (event, index, reportStatus) => {
+    this.setState({
+      reportStatus: reportStatus
+    })
+  }
+
+  handleReportSortClicked = (event, index, value) => {
+    this.setState({
+      sortedReports: value
+    })
+  }
+
+  reportFilterMenu (reportStatus) {
+    const status = ['Draft', 'Finalized', 'Created By Me']
+    return status.map((name) => (
+      <MenuItem
+        key={name}
+        insetChildren
+        checked={reportStatus && reportStatus.indexOf(name) > -1}
+        value={name}
+        primaryText={name}
+      />
+    ))
+  }
+
+  reportSortMenu (sortedReports) {
+    const dateAdded = ['Newest', 'Oldest', 'Name (A to Z)', 'Name (Z to A)']
+    return dateAdded.map((elem, index) => (
+      <MenuItem
+        key={elem}
+        value={index}
+        primaryText={elem}
+      />
+    ))
+  }
+
+  filteredReportItems () {
+    const { reportList, currentUser } = this.props
+    const { reportStatus, sortedReports } = this.state
+    const statusFilter = reportStatus.length === 3 || (reportStatus.includes('Draft') && reportStatus.includes('Finalized')) ? report => true
+      : reportStatus.includes('Draft') ? report => report.status === 'UNCONFIRMED'
+        : reportStatus.includes('Finalized') ? report => report.status === 'CONFIRMED' : report => true
```

Comment:
Change the check to `report.status !== REPORT_DRAFT_STATUS`.
We should no longer use 'CONFIRMED' for determining what the report status is currently (see #493 )